packages/plugin-core/src/components/lookup/LookupProviderV3.ts

```diff
@@ -608,7 +609,19 @@ export class SchemaLookupProvider implements ILookupProviderV3 {
       }
       const isMultiLevel = picker.value.split(".").length > 1;
       if (isMultiLevel) {
-        window.showErrorMessage("schemas can only be one level deep");
+        window
+          .showInformationMessage(
+            "It looks like you are trying to create a multi-level [schema](https://wiki.dendron.so/notes/c5e5adde-5459-409b-b34d-a0d75cbb1052.html). This is not supported. If you are trying to create a note instead, run the `> Note Lookup` command or click on `Note Lookup`",
+            ...["Note Lookup"]
+          )
+          .then(async (selection) => {
+            if (selection === "Note Lookup") {
+              await new NoteLookupCommand().run({
+                initialValue: picker.value,
+              });
+            }
+          });
+        //window.showErrorMessage("schemas can only be one level deep");
```

Comment:
if you remove the `return` value, this will work. the reason its hanging is because of the following:

<img width="1677" alt="Note_Preview_—_dendron__Workspace_" src="https://user-images.githubusercontent.com/409321/146469602-3219066d-806c-472b-8026-d36a3ed84504.png">

Docs: [[Initialization|dendron://dendron.docs/pkg.plugin-core.t.lookup.arch#initialization]]

Essentially, that command only returns when it receives an event from the `HistoryService` which is never done in this case. This is actually a bug in our code.